.env.production

```
@@ -0,0 +1,3 @@
+NODE_ENV=development
+DEV_PROD_DOMAIN=
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The DEV_PROD_DOMAIN key should go before the NODE_ENV key

(UnorderedKey)